src/main/java/cofh/thermaldynamics/duct/fluid/DuctUnitFluid.java

```diff
@@ -501,6 +501,15 @@ public void onPlaced(EntityLivingBase living, ItemStack stack) {
 		}
 	}
 
+	@Override
```

Comment:
context: super class returns false be default. The servo needs this to return true, otherwise the tile wont be cached as an input tile for the grid.